config/workbox-cli-config.js

```diff
@@ -0,0 +1,36 @@
+const paths = require('./paths.json') // specify paths to main working directories
+
+module.exports = {
+  globDirectory: paths.public,
+  globPatterns: [
+    '**/*.{html,js,css,png,jpg}'
+  ],
+  swDest: paths.public + 'service-worker.js',
+  importWorkboxFrom: 'local',
+  skipWaiting: true,
```

Comment:
Sorry didnt notice this earlier, should we have skipWaiting and clientsClaim here? I was under the impression these are used in development and break out of the service worker lifecycle?

Reply:
Think this may be OK since we're doing a network first approach

Reply:
The waiting phase is useful when you're only running one version of your site at once, but since we don't need that feature we can make the new service worker activate sooner, so I think setting it to `true` is good.